### DISCUSSION TITLE
Recognition annotations confusion

### DISCUSSION THREAD

**Discussion by Enverrrr** [open-mmlab/mmocr#1690] [Q&A]
Hey I want to train SAR recognition model to read license plate. I prepared a dataset and annotated it after that I trained on 'sar_r31_parallel_decoder_toy_dataset.py' however the model print just first part of license plate here an example
![oç_mmocr](https://user-images.githubusercontent.com/60448321/214792145-85b6b297-e0ea-4025-89cf-51ff45687470.png)

and here annotation file
[boşluk_label.txt](https://github.com/open-mmlab/mmocr/files/10507585/bosluk_label.txt)
also I tried without space among the part of text
[label.txt](https://github.com/open-mmlab/mmocr/files/10507538/label.txt)

Unfortunately it's not working. In my opinion I annotated wrong way but I'm not sure what can I do?
P.S. I turned of detection algorithm in mmocr/utils/ocr.py

**Comment by gaotongxiao** [comment]
>  I tried without space among the part of text [label.txt](https://github.com/open-mmlab/mmocr/files/10507538/label.txt)

I think what you did was right. How did the model perform with this annotation?
